### ISSUE TITLE
About whole download

### ISSUE THREAD

**Issue by rabbitrun00** [xsthunder/download-youtube-subtitle#15]
Can this tool whole download cc caption tranlation in playlist?

**Comment by xsthunder** [comment]
Yes on version 3.0.2. We introduced `dl-youtube-cc-playlist`. Please check https://github.com/xsthunder/download-youtube-subtitle#download-the-caption-of-entire-playlist

You may find a online version on colab. please check https://colab.research.google.com/drive/1oseD2yEsScx0YYOZ1x1F8GSG9iJ4x3qi?usp=sharing#scrollTo=qstfXMbzG62N
